Makefile

```diff
@@ -5,7 +5,7 @@ BINDIR=node_modules/.bin
 JSHINT=$(BINDIR)/jshint
 ESLINT=$(BINDIR)/eslint
 JSCS=$(BINDIR)/jscs
-MOCHA=$(BINDIR)/_mocha
+MOCHA=$(BINDIR)/mocha
```

Comment:
Something happened with `mocha` at `2.2.0` and `_mocha` doesn't work anymore. Everything else seems to work with `mocha`, but I encourage you to check before merging.